lib/record_store/zone.rb

```diff
@@ -116,7 +116,7 @@ def build_records(records)
 
     def validate_records
       records.each do |record|
-        unless record.fqdn.end_with?(name)
+        unless record.fqdn.ends_with?(name)
```

Comment:
`ends_with?` doesn't exist in the Ruby standard library, but does in `activesupport`.

This change depends on the transitive dependency:

`oci -> activesupport`

...we may want to explicitly require `activesupport` so there're no surprises if `oci` was removed or replaced by another library.

Also, it feels like these sorts of changes should happen in separate PRs to keep this one focused.  There are a tonne of opportunities to use `activesupport` now that we need to depend on it.

Reply:
But why do we need this ? `end_with?` is plenty sufficient for this... and as showed with the previous link the ActiveSupport version of it has been deprecated ?!

Reply:
Agreed.  Even approaching refactoring for `activesupport` doesn't need to happen here.  It's tempting as heck, but...should be thought through a little more.